packages/python-pulp-cli-deb/python-pulp-cli-deb.spec

```
@@ -0,0 +1,68 @@
+%{?scl:%scl_package python-%{pypi_name}}
+%{!?scl:%global pkg_name %{name}}
+
+# Created by pyp2rpm-3.3.7
+%global pypi_name pulp-cli-deb
+
+Name:           %{?scl_prefix}python-%{pypi_name}
+Version:        0.0.1 
+Release:        1%{?dist}
+Summary:        Command line interface to talk to pulpcore's REST API
+
+License:        GPLv2+
+URL:            https://github.com/pulp/pulp-cli-deb
+Source0:        https://files.pythonhosted.org/packages/source/p/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
+BuildArch:      noarch
+
+BuildRequires:  %{?scl_prefix}python3-devel
+BuildRequires:  %{?scl_prefix}python3-setuptools
```

```suggestion
BuildRequires:  %{?scl_prefix}python%{python3_pkgversion}-devel
BuildRequires:  %{?scl_prefix}python%{python3_pkgversion}-setuptools
```
Copied from the `python-pulp-cli.spec` file. There are many more instances of `python3` in this spec file, that should be similarly adjusted (probably).